.github/workflows/lint.yml

```
@@ -0,0 +1,21 @@
+name: Lint
+
+on:
+  push:
+
+jobs:
+  check-format:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+      
+      - name: Setup Node
+        uses: actions/setup-node@v4
+
+      - name: Install dependencies
+        run: npm ci
+  
+      - name: Fix formatting
+        run: npx prettier . --write
```

```suggestion
        run: npx prettier . --check
```
Please don't try to create a format workflow. Just run `npm run prettier:fix` locally and commit the result